app/views/layouts/application.html.erb

```diff
@@ -18,7 +18,11 @@
                 <%= link_to "New Game", new_game_path, class: "nav-link" %>
               </li>
               <li class="nav-item">
-                  <a class="nav-link" href="#">Concede</a>
+<% if false %>
```

Comment:
Added new Concede button. It actually does work as intended until the game gets deleted. Once the page loads without a current game in progress, this code will initiate an error saying invalid game.id. I'm leaving this commented out with the "<% if false %> .... <% end %>" lines. If you want to test this button and help me fix the error, just delete lines 21 and 24.